pulsar-client-cpp-version.txt

```diff
@@ -0,0 +1 @@
+3.0.0
```

Comment:
Where are we going to define the version of the wrapper?

Reply:
In pakcage.json -> version

Reply:
👍